general/src/protocol/contexts.rs

```diff
@@ -336,6 +342,13 @@ pub struct TraceContext {
     /// The ID of the span.
     pub span_id: Annotated<SpanId>,
 
+    /// The ID of the span enclosing this span.
+    pub parent_span_id: Annotated<SpanId>,
```

Comment:
Is it too late to name this shorter? It’s kinda strange that this has such a long name, but `op` is not typed out at the same time.

Reply:
That's true but it's quite engrained by now. I think we could rename it in semaphore if we needed to.